### DISCUSSION TITLE
limiting the number of selected items for multi select picker input

### DISCUSSION THREAD

**Discussion by mamuna36** [epam/UUI#511] [Q&A] [answered]
Is there an option to limit the number of the selected item in the multi-select mode of picker input?

**Comment by AlekseyManetov** [comment, marked as answer]
Hi, you can make it by disabling row checkbox via `getRowOptions` prop - https://codesandbox.io/s/uui-forked-bwb3z2?file=/Example.tsx

P.s. There is an error when you try to focus disabled row, we will fix it in feature version.

**Comment by mamuna36** [comment]
thanks for your quick response. This means that we can not use this option for now. How long would it take to release this new feature?

**Comment by AlekseyManetov** [comment]
We plan next release at the end of current week or at the beginning of next week.
 Also you can try workaround when you move this limit condition to the onValueChange callback: 
`(newVal) => {  setValue(newValue.length > 3 ? currentValue : newValue) }`
 but this approach do not provide such good UX like with getRowOptions, you can use it like a temporary solution
